app/src/main/AndroidManifest.xml

```diff
@@ -10,17 +10,23 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
-        <activity android:name=".MainActivity">
+        <activity android:name="com.brandio.androidsample.ShowListWithBannerActivity"></activity>
+        <activity android:name="com.brandio.androidsample.MainActivity">
```

Comment:
Why using the full path (com.brandio.androidsample.MainActivity) instead of .MainActivity?

Reply:
It is not necessary, will change it